main.nf

```diff
@@ -422,22 +423,21 @@ workflow {
     bam_mapped = bam_single.mix(MERGE_BAM_MAPPED(bam_multiple))
     //if(save_bam_mapped || !(params.known_indels))
     //TODO: https://github.com/nf-core/sarek/blob/bce378e09de25bb26c388b917f93f84806d3ba27/main.nf#L1478
-    //But  if SAMTOOLS_INDEX is not run, markduplicates does not work
-         bam_mapped = SAMTOOLS_INDEX_MAPPED(bam_mapped)
+    //But if SAMTOOLS_INDEX is not run, markduplicates does not work
+    bam_mapped = SAMTOOLS_INDEX_MAPPED(bam_mapped)
```

Comment:
This comment is obsolet with your fix, right?

Reply:
Wasn't sure about it, but probably